### ISSUE TITLE
Cannot install Roller Coaster Tycoon (GOG)

### ISSUE THREAD

**Issue by haffmans** [Heroic-Games-Launcher/HeroicGamesLauncher#2112]
### Describe the bug

When trying to install Roller Coaster Tycoon from GOG, the dialog keeps spinning with the "Geting download size..." and "Geting install size..." 

The log shows it's trying to fetch the info:
```
(23:16:28) INFO:    [Gog]:              Getting game metadata: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1207658945 --token <redacted> --lang=en-US --os windows
(23:16:28) INFO:    [Gog]:              Getting game metadata: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1207658945 --token <redacted> --lang=en-US --os windows
```

The output of this command is a download and disk size of 0:

```
API] INFO: Getting info from products endpoint for id: 1207658945
[DOWNLOAD_MANAGER] INFO: Checking compatibility of Roller Coaster Tycoon® Deluxe with windows
[DOWNLOAD_MANAGER] INFO: Game is compatible
[DOWNLOAD_MANAGER] INFO: Depot version: 2
[API] INFO: Getting Dependencies repository
[API] INFO: Getting repository manifest
[DOWNLOAD_MANAGER] INFO: Calculating download size
{"download_size": 0, "disk_size": 0, "dlcs": [], "buildId": "50860644332681884", "languages": ["de-de", "en-us", "es-es", "fr-fr", "it-it"], "folder_name": "RollerCoaster Tycoon Deluxe", "versionEtag": "\"4159993833\"", "versionName": "1.20.015"}
```

Changing the language argument of gogdl to `en-us` (lowercase), the result works:

```
[API] INFO: Getting info from products endpoint for id: 1207658945
[DOWNLOAD_MANAGER] INFO: Checking compatibility of Roller Coaster Tycoon® Deluxe with windows
[DOWNLOAD_MANAGER] INFO: Game is compatible
[DOWNLOAD_MANAGER] INFO: Depot version: 2
[API] INFO: Getting Dependencies repository
[API] INFO: Getting repository manifest
[DOWNLOAD_MANAGER] INFO: Calculating download size
{"download_size": 410376713, "disk_size": 537532740, "dlcs": [], "buildId": "50860644332681884", "languages": ["de-de", "en-us", "es-es", "fr-fr", "it-it"], "folder_name": "RollerCoaster Tycoon Deluxe", "versionEtag": "\"4159993833\"", "versionName": "1.20.015"}
```

### Add logs

```text
(23:14:45) INFO:    [Connection]:       Connectivity: check-online
(23:14:45) INFO:    [Connection]:       Pinging external endpoints
(23:14:45) INFO:    [Legendary]:        Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(23:14:45) INFO:    [Legendary]:        Legendary location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(23:14:45) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(23:14:45) WARNING: [Backend]:          Protocol already registered.
(23:14:45) INFO:    [Connection]:       Connectivity: online
(23:14:45) INFO:    [Gog]:              Getting data about the user
(23:14:45) INFO:    [Gog]:              Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --version
(23:14:45) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(23:14:45) INFO:    [Backend]:          
Heroic Version: 2.5.0 Yamato
Legendary Version:  0.20.31 Dark Energy (hotfix #5)
GOGdl Version: 0.5
OS: Arch KERNEL: 6.0.10-arch2-1 ARCH: x64
CPU: AMD Ryzen 7 5700X 8-Core Processor @3.78 GOVERNOR: schedutil
RAM: Total: 31.25 GiB Available: 22.45 GiB
GRAPHICS: GPU0: Navi 22 [Radeon RX 6700/6700 XT/6750 XT / 6800M] VRAM: 1MB
PROTOCOL: x11
(23:14:45) INFO:    [Gog]:              Saved user data to config
<snip>
(23:16:28) INFO:    [Gog]:              Getting game metadata: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1207658945 --token <redacted> --lang=en-US --os windows
(23:16:28) INFO:    [Gog]:              Getting game metadata: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1207658945 --token <redacted> --lang=en-US --os windows
(23:16:28) DEBUG:   [Backend]:          {"free":63782535168,"diskSize":497925136384,"message":"59.4 GiB / 463.73 GiB","validPath":true}
(23:16:28) DEBUG:   [Backend]:          {"free":63782535168,"diskSize":497925136384,"message":"59.4 GiB / 463.73 GiB","validPath":true}
```

### Steps to reproduce

1. Get the GOG version of RollerCoaster Tycoon
2. Click on the Install button
3. Optionally also change the selected language in the install box

### Expected behavior

Size data should be fetched and presented correctly, and the install should be able to complete.

### Screenshots

![image](https://user-images.githubusercontent.com/2103323/204921290-a12584f5-8b01-4f65-a9d8-fcf1291314fe.png)

### System Information

- OS [e. g. "Ubuntu"]: ArchLinux
- Heroic Version [e. g. 2.1.1]: 2.5.0 Yamato

**Comment by flavioislima** [comment]
@haffmans we are working on a fix for that. Can you try this appImage to see if it installs just fine?
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/actions/runs/3586149679

**Comment by haffmans** [comment]
Unfortunately that version doesn't seem to solve it, I'm still seeing the same symptoms as before.

```
(00:32:32) INFO:    [Connection]:       Connectivity: check-online
(00:32:32) INFO:    [Connection]:       Pinging external endpoints
(00:32:32) INFO:    [Legendary]:        Running command: /tmp/.mount_HeroicEl6rRM/resources/app.asar.unpacked/build/bin/linux/legendary --version
(00:32:32) INFO:    [Legendary]:        Legendary location: /tmp/.mount_HeroicEl6rRM/resources/app.asar.unpacked/build/bin/linux/legendary
(00:32:32) INFO:    [Gog]:              GOGDL location: /home/cheetah/bin/gogdl.sh
(00:32:33) WARNING: [Backend]:          Protocol already registered.
(00:32:33) INFO:    [Connection]:       Connectivity: online
(00:32:33) INFO:    [Gog]:              Getting data about the user
(00:32:33) INFO:    [Gog]:              Running command: /home/cheetah/bin/gogdl.sh --version
(00:32:33) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(00:32:33) INFO:    [Backend]:          
Heroic Version: 2.5.0 Yamato
Legendary Version:  0.20.31 Dark Energy (hotfix #5)
GOGdl Version: 0.5
OS: Arch KERNEL: 6.0.10-arch2-1 ARCH: x64
CPU: AMD Ryzen 7 5700X 8-Core Processor @3.67 GOVERNOR: schedutil
RAM: Total: 31.25 GiB Available: 23.13 GiB
GRAPHICS: GPU0: Navi 22 [Radeon RX 6700/6700 XT/6750 XT / 6800M] VRAM: 1MB  
PROTOCOL: x11
<snip>
(00:33:04) INFO:    [Gog]:              Getting game metadata: /tmp/.mount_HeroicEl6rRM/resources/app.asar.unpacked/build/bin/linux/gogdl info 1207658945 --token <redacted> --lang=en-US --os windows
(00:33:04) INFO:    [Gog]:              Getting game metadata: /tmp/.mount_HeroicEl6rRM/resources/app.asar.unpacked/build/bin/linux/gogdl info 1207658945 --token <redacted> --lang=en-US --os windows
(00:33:05) DEBUG:   [Backend]:          {"free":54705901568,"diskSize":497925136384,"message":"50.95 GiB / 463.73 GiB","validPath":true}
(00:33:05) INFO:    [Gog]:              Getting game metadata: /tmp/.mount_HeroicEl6rRM/resources/app.asar.unpacked/build/bin/linux/gogdl info 1207658945 --token <redacted> --lang=en-US --os windows
(00:33:05) DEBUG:   [Backend]:          {"free":54705901568,"diskSize":497925136384,"message":"50.95 GiB / 463.73 GiB","validPath":true}
```

**Comment by flavioislima** [comment]
ok, then it is another issue we are fixing, check this other build:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/actions/runs/3586149679
